apps/codesandbox-react-next-template/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": ["@uifabric/tslint-rules"],
+  "rules": {
+    "deprecation": false,
+    "no-any": false,
+    "prefer-const": false
```

Comment:
None of these should be disabled for demo code, which ought to show best practices. Same in other template.